### PR TITLE
feat: Add additional static macros

### PR DESCRIPTION
**Pull request by alex-barstow**
## Description
Fairly straightforward additions to the static macros

### CODE REVIEW

File: docs/integrator/macros.md [hunk 1]
```diff
@@ -41,8 +41,12 @@ Replaces macros in a given string with their corresponding values.
 | {player.widthInt}        | The current player width as int       |
 | {player.heightInt}       | The current player height as int      |
 | {player.duration}        | The duration of current video *       |
-| {player.durationInt}     | The duration as int *                  |
+| {player.durationInt}     | The duration as int *                 |
```
Comment (usmanonazim) [inline docs/integrator/macros.md:44]: what is the purpose of the '*' at the end of some of the macro descriptions? is that just for padding?

**Review by usmanonazim — APPROVED** [review]
LGTM

**Review by Essk — APPROVED** [review]

File: docs/integrator/macros.md (hunk 1 above)
Comment (Essk) [reply docs/integrator/macros.md:44]: it's referencing a note / qualification on line 65:
> \* Returns 0 if video is not loaded. Be careful timing your ad request with this macro.

Comment (usmanonazim) [reply docs/integrator/macros.md:44]: Thank you 😁

**Pull request merged**